datamade/committee-oversight
# Add Congresses to model admin and implement footnotes

## Overview

Closes #178. This PR adds the ability to create footnotes for each Congress on the site, and adds them to the templates following Jamie's request below. Also, while adding Congress to the ModelAdmin, I've added a more user-friendly way to adjust the `inactive_days` value for each Congress (as discussed in #176)

From Jamie, here are the 3 places footnotes should show up:

1. Next to the congress on individual committee pages (footnote should be placed directly under chart)
![unnamed](https://user-images.githubusercontent.com/1094243/80142220-712fa280-8570-11ea-962f-dfd0c45db0dc.jpg)

2. Next to the “Projected Grade” on individual committee pages (footnote should be placed directly under chart) 
![unnamed (1)](https://user-images.githubusercontent.com/1094243/80142238-755bc000-8570-11ea-8b1e-77d66dbf05d7.jpg)

3. Next to Current Congress (projected) on landing page (footnote should be at bottom of page)
![unnamed (2)](https://user-images.githubusercontent.com/1094243/80142263-7d1b6480-8570-11ea-8044-b40e134d64df.jpg)

## Testing Instructions

 * If you haven't recently, download an updated db from production [here](https://s3.console.aws.amazon.com/s3/object/datamade-postgresql-backups/hearings/%255C2020%255C04%255C23%255C16%255C30.dump?region=us-east-1). Add it to the root of your project as `hearings.dump`
* `docker-compose -f docker-compose.yml -f docker-compose.db-ops.yml run -e PGPASSWORD=postgres --rm dbload-dump`
* `docker-compose run --rm app python manage.py migrate`
* `docker-compose up`
* Log into Wagtail and go to the "Edit Congresses" section of the left sidebar. Add one and then multiple footnote values  and make sure they shows up on the site as described above
